css/index.css

```diff
@@ -143,4 +239,133 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
-}
\ No newline at end of file
+}
+
+/* ABOUT PAGE CSS */
+hr {
+    border: 1px dashed;
+}
+
+hr.about {
+    width: 790px;
+}
+
+.about-text {
+    width: 700px;
+    margin: 20px auto;
+}
+
+.container.about-page {
+    width: 700px;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+}
+
+.content-box img {
+    width: 100%;
+}
+
+.content-box {
+    margin: 20px 0;
+    width: 48%;
+}
+
+.content-box h3, .content-box p {
+    margin: 10px 0;
+}
+
+button {
+    cursor: pointer;
+}
+
+.learn-more-btn {
+  background-color: white;
+  border: 2px solid black;
+  border-radius: 10px;
+  color: black;
+  padding: 8px 22px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 14px;
+  margin: 10px 0;
+  transition: 0.5s;
+}
+
+.learn-more-btn:hover {
+   background-color: black;
+   border: 2px solid white;
+   color: white; 
+   border-radius: 0;
+}
+
+.lets-work-together {
+    margin: 30px 0;
+}
+
+.about-page-footer {
+    width: 800px;
+    margin: auto;
+}
+
+/* CONTACT PAGE CSS */
+.contact-us-container {
+    width: 800px;
+    margin: auto;
+    text-align: center;
+}
+
+.contact-us-container h1 {
+    font-size: 32px;
+    font-weight: 400;
+}
+
+.contact-form {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.contact-form .text-input {
+    width: 50%;
+    padding: 10px;
+    border-radius: 5px;
+    border: 1px solid rgb(201, 201, 201);
+    color: black;
+    margin: 10px;
+    
+}
+
+textarea {
+    height: 130px;
+    width: 50%;
+    padding: 0;
+    resize: none;
+    border-radius: 5px;
+    border: 1px solid rgb(201, 201, 201);
+    color: black;
+}
+
+.submit-btn {
```

Comment:
again, LESS will DRY this up a lot and I think you will enjoy using it after we look at it this week